test/gov/gov-combined.sh

```diff
@@ -149,135 +149,260 @@ if [ "${EXCH_APP_HOST}" = "http://exchange-api:8080/v1" ]; then
 else
 
   cd /root
+  if [ ${CERT_LOC} -eq "1" ]; then
+    echo "Delete e2edev@somecomp.com..."
```

Comment:
I think you need to put both side of this if then else into a script that you can condition with a parameter. There's just too much duplication here now.

Reply:
I agree with your statement, I will work on something. I think the majority of the if statements are there to either add the --cert or remove it in the curl command.